changelog/_unreleased/2021-01-14-add-missing-require-flags-in-sw-customer.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Add missing require flags in sw-customer
+issue: NEXT-12173
+author: Jonas Dambacher
+author_email: j.dambacher@basecom.de
+author_github: jdambacher
+---
+___
```

Comment:
This line needs to be removed otherwise our changelog parser can´t parse the file :)